test/functional/system/0.0.transfer.js

```diff
@@ -28,6 +28,7 @@ describe('system test (type 0) - double transfers', () => {
 
 	var i = 0;
 	var t = 0;
+	/* eslint-disable no-loop-func */
```

Comment:
This while loop is very strange. Do we need it?

Reply:
@willclarktech I think @diego-G will be able to answer this better, as I have no clue why we need to loop here for 30 times in specific or looping logic necessity.

Reply:
These loops were useful to find bugs related to 0.9.11 & 0.9.12 patch. I suggest to keep them until porting 0.9.12.

Reply:
@diego-G The difference for this particular loop is it only runs once. Do you know why?

Reply:
All those kind of loops from system test suite are doing the same thing through different type of transactions. I just wanted to maintain them there as a handy for development and testing. We can remove it later on.